### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) combines these small changes PRs: #791, #793, and #794 

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

### CODE REVIEW

File: src/catalog/inventory-management.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ sections:
...
+Magento Inventory Management gives you the tools to manage your product inventory. Merchants with a single store to multiple warehouses, stores, pickup locations, drop shippers, and more can use these features to maintain quantities for sales and handle shipments to complete orders. Track your inventory quantities, provide accurate salable stock amounts to customers for all of your websites, and ship according to recommendations based on distance or priority. And configure your preferred product configurations globally (for all stores and products), per source, and per product. These features grow with your business, allowing you to work from a single warehouse or complex shipping network with a few additional settings.
```
Comment (hickskenh) [inline src/catalog/inventory-management.md:33]:
```suggestion
Magento Inventory Management gives you the tools to manage your product inventory. Merchants with a single store to multiple warehouses, stores, pickup locations, drop shippers, and more can use these features to maintain quantities for sales and handle shipments to complete orders. You can track your inventory quantities, provide accurate salable stock amounts to customers for all of your websites, and ship according to recommendations based on distance or priority. You can also configure your preferred product configurations globally (for all stores and products), per source, and per product. These features grow with your business, allowing you to work from a single warehouse or complex shipping network with a few additional settings.
```

File: src/sales/quote-workflow.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Step 1: Buyer requests quote
...
 
 Step 2: Seller views request and sends response
-: The seller views the request for a quote. The status of the quote changes to `Pending`, and they buyer cannot make any changes. The [seller responds]({% link sales/quote-price-negotiation.md %}) by offering discounted pricing for the products in the quote, enters a comment , and sends the quote back to the buyer. The buyer and sales representative are notified by email that the seller has responded.
+: The seller views the request for a quote. The status of the quote changes to `Pending`, and the buyer cannot make any changes. The [seller responds]({% link sales/quote-price-negotiation.md %}) by offering discounted pricing for the products in the quote, enters a comment , and sends the quote back to the buyer. The buyer and sales representative are notified by email that the seller has responded.
```
Comment (hickskenh) [inline src/sales/quote-workflow.md:12]:
```suggestion
: The seller views the request for a quote. The status of the quote changes to `Pending`, and the buyer cannot make any changes. The [seller responds]({% link sales/quote-price-negotiation.md %}) by offering discounted pricing for the products in the quote, enters a comment, and sends the quote back to the buyer. The buyer and sales representative are notified by email that the seller has responded.
```

**Review by hickskenh — APPROVED** [review]
approved after your review of suggestions

**Comment by leslietilling** [comment]
running tests

**Pull request merged**